tst/regression/test_suites/advection_convergence/advection_convergence.py

```diff
@@ -66,7 +66,7 @@ def Prepare(self,parameters, step):
             if lin_res[step - 1] == 32:
                 parameters.coverage_status = "both"
             parameters.driver_cmd_line_args = [
-                'parthenon/mesh/nx1=%d' % lin_res[step % n_res -1],
+                'parthenon/mesh/nx1=%d' % (lin_res[step % n_res -1] * parameters.num_ranks),
```

Comment:
This is cool. But do we actually want to do a scaling test and a convergence test in one? This could get ludicrously expensive, depending on what we're doing?

Or am I misunderstanding what's going on here?

Reply:
Maybe weak scaling is the wrong word choice. The point is to be able to run the tests using more than a single mpi processor. The number of mpi procs for the tests is set at compile time... (for the tests) and we are only running them once. 

Weak scaling would imply that I'm running each tests using several different numbers of mpi procs this is not true... so my choice of weak scaling is ... misleading. In theory you could do this but like you said we would be using a lot of resources. The point is we now have the **flexibility** to run the tests with different numbers of gpus and mpi procs. 

Before this we were limited essentially to running the tests with only a single mpi proc and a single gpu. When I update the documentation I would appreciate your feedback to be sure this is clear.

Reply:
Ah, ok. I understand. I agree, this flexibility is desirable. Yeah weak scaling means something specific in my brain, so I got confused.

> When I update the documentation I would appreciate your feedback to be sure this is clear.

Sounds good.